tests/chart_tests/test_houston_ingress.py

```diff
@@ -72,3 +72,29 @@ def test_protect_houston_internal_urls(self, kube_version):
 }
 """
         )
+
+    def test_houston_ingress_overrides(self, kube_version):
+        custom_annotations = {
+            "nginx.ingress.kubernetes.io/upstream-keepalive-connections": "400",
+            "nginx.ingress.kubernetes.io/upstream-keepalive-timeout": "120",
```

Comment:
Let's change these to something conspicuous:
```suggestion
            "nginx.ingress.kubernetes.io/upstream-keepalive-connections": "999",
            "nginx.ingress.kubernetes.io/upstream-keepalive-timeout": "777",
```